code/index.html

```diff
@@ -4,27 +4,88 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="style.css">
   </head>
-  <body>
+    <body>
+
     <!-- video or image as a header is cool :) -->
+      <div class="hero-image-container">
+        <div class="hero-title-container">
+          <h1><span>Pura Vida travels</span></h1>
+          <h2>Tailor-made trips to Costa Rica</h2>
+        </div>
+      </div>
+      
+    <!-- Wrap accordion and signup with flexbox -->
+
+  <div class="wrapper">
+
+    <!--Sign up -->
+    
+      <section class="contact-form">
+        <form action="https://httpbin.org/anything" method="POST">
+          <h1>Contact us</h1>
+
+          <div class="textfield">
+            <label>Full name</label>
+            <input type="text" name="" value="" placeholder="Write your name">
+          </div>
+
+          <div class="textfield">
+            <label>Email</label>
+            <input type="text" name="" value="" placeholder="Write your email">
```

Comment:
There's a type called email that you can use here! And if you don't want a name or value on your inputs - omit them instead of leaving them blank

Reply:
Although it's good to give them names to easier be able to read the submission of the form :)